templates/uefi/generic-qemu-uefi-boot-template.jinja2

```diff
@@ -0,0 +1,72 @@
+{% extends 'base/kernel-ci-base.jinja2' %}
+{% set ctx_arch = arch %}
+{% if arch == "arm" %}
+{% set ctx_arch = 'arm64' %}
+{% set ctx_cpu = 'cortex-a15' %}
+{% set console_dev = 'ttyAMA0' %}
+{% set image_url_uefi = 'http://people.linaro.org/~ard.biesheuvel/kernelci/QEMU_EFI.fd-ARM-RELEASE-325ad6226099' %}
```

Comment:
We probably need to host these files somewhere on storage.kernelci.org, yea?

Reply:
I'm not convinced, there's both pros and cons...
But please convince me =)

Reply:
I think this should be a parameter defined in `test-configs.yaml`, which is a site-specific config file unlike test templates.

Reply:
I think we should start using Ard's firmware binaries and put them onto storage.kernelci.org/uefi/...
Today Linaro does releases for arm and aarch64 not for x64 or ia32 [1]. However, I got promised that they will add those to the next release that will happen 19.08. On the release page there is documented how stuff is built.
If  we just stick a file next to the storage.kernelci.org/uefi/19.08/build-info.txt and pint to the release page or something that should be enough for reproducing the build if needed.

[1] http://releases.linaro.org/reference-platform/enterprise/firmware/open-source/19.03/